qemu/tests/blockdev_snapshot_multi_disks.py

```diff
@@ -0,0 +1,139 @@
+        for snapshot_tag in self.snapshot_tag_list:
+            if self.is_blockdev_mode():
+                params = self.params.copy()
+                params.setdefault("target_path", data_dir.get_data_dir())
+                image = sp_admin.volume_define_by_params(snapshot_tag, params)
+                image.hotplug(self.main_vm)
+            else:
+                if self.params.get("mode") == "existing":
+                    snapshot_image = self.get_image_by_tag(snapshot_tag)
+                    snapshot_image.create()
+
+    @error_context.context_aware
+    def create_snapshot(self):
+        error_context.context("do snaoshot on multi_disks",
+                              logging.info)
+        assert len(
+            self.target_disks) == len(
+            self.source_disks), "No enough target disks define in cfg!"
+        source_lst = list(map(lambda x: "drive_%s" % x, self.source_disks))
+        target_lst = list(map(lambda x: "drive_%s" % x, self.target_disks))
+        arguments = {}
+        if len(source_lst) > 1:
+            error_context.context(
+                "snapshot %s to %s " % (source_lst, target_lst))
+            backup_utils.blockdev_batch_snapshot(
+                self.main_vm, source_lst, target_lst, **arguments)
+        else:
+            error_context.context(
+                "snapshot %s to %s" %
+                (source_lst[0], target_lst[0]))
+            backup_utils.blockdev_snapshot(
+                self.main_vm,
+                source_lst[0],
+                target_lst[0])
+
+    def verify_snapshot(self):
+        if self.main_vm.is_alive():
+            self.main_vm.destroy()
+        for snapshot_tag, base_tag in zip(self.snapshot_tag_list,
+                                          self.base_tag_list):
+            if self.is_blockdev_mode():
+                snapshot_image = self.get_image_by_tag(snapshot_tag)
+                base_image = self.get_image_by_tag(base_tag)
+                snapshot_image.base_tag = base_tag
+                snapshot_image.base_format = base_image.get_format()
+                base_image_filename = base_image.image_filename
+                snapshot_image.base_image_filename = base_image_filename
+                snapshot_image.rebase(snapshot_image.params)
+        self.clone_vm.create()
+        self.clone_vm.verify_alive()
+        if self.base_tag != "image1":
+            self.mount_data_disks()
+            self.verify_data_file()
+
+    def post_test(self):
+        try:
+            self.clone_vm.destroy()
+            for snapshot_tag in self.snapshot_list:
+                snapshot_image = self.get_image_by_tag(snapshot_tag)
+                snapshot_image.remove()
+        except Exception as error:
+            logging.error(str(error))
+
+
+def run(test, params, env):
+    """
+    Backup VM disk test when VM reboot
+
+    1) start VM with system disk
+    2) create target disk with qmp command
+    3) load stress in guest
```

Comment:
where step to load stress?

Reply:
Hi, have updated, please help to review again.